README.md

```diff
@@ -77,6 +77,8 @@ You can build ChakraCore on Windows 7 SP1 or above, and Windows Server 2008 R2 o
 
 More details in [Building ChakraCore](https://github.com/Microsoft/ChakraCore/wiki/Building-ChakraCore).
 
+Alternatively, you can get pre-built ChakraCore binaries on [myget](https://www.myget.org/feed/chakracore-preview/package/nuget/Microsoft.ChakraCore).  
```

Comment:
In light of https://github.com/Microsoft/ChakraCore-wiki/pull/3 recommend changing this to:

`Alternatively, you can get pre-built ChakraCore binaries from our [NuGet Packages](https://github.com/Microsoft/ChakraCore/wiki/NuGet-Packages).`